.travis.yml

```diff
@@ -43,21 +48,25 @@ script:
   - npm run lint
   - "if [ $COVERAGE ]; then npm run cover && bash <(curl -s https://codecov.io/bash) -f coverage/lcov.info; else npm run test; fi"
 
-after_success:
-  - npm run semantic-release
-
-branches:
-  except:
-    - /^v\d+\.\d+\.\d+$/
-
-deploy:
-  provider: surge
-  project: ./esdoc/
-  domain: docs.sequelizejs.com
-  skip_cleanup: true
-  on:
-    branch: master
-    node: 6
-
-before_deploy:
-  - npm run docs
+jobs:
+  include:
+    - stage: docs
+      node_js: 6
+      script:
+        - npm run lint
+        - npm run docs
+      deploy:
+        provider: surge
+        project: ./esdoc/
+        domain: docs.sequelizejs.com
+        skip_cleanup: true
+        on:
+          branch: master
+    - stage: release
+      node_js: 4
+      script: npm run lint
+      deploy:
+        provider: script
+        script: TRAVIS_JOB_NUMBER=WORKAROUND.1 npm run semantic-release
```

Comment:
I never personally checked if this failed @felixfbecker , during my study for build stages I found https://github.com/semantic-release/semantic-release/issues/390 issue indicating it might., so I just included it just to be safe